### ISSUE TITLE
[MHV SM / Accessibility] Input error message not announced

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#124531]
### 📝 User story
As a screen reader user, I want form errors to be announced, so that I know there are errors present that prevent progress and I can address them.

### 📝 Problem statement
Selecting Continue with incomplete inputs causes an error to display but the error message is not announced by the screen reader.

### 📝 Details
<details><summary>Links, screenshots, steps to recreate</summary>

#### 🌌 Supporting screenshots, videos, code snippets

---

##### 🔗 From [Secure Messages > Care teams you recently sent messages to](https://staging.va.gov/my-health/secure-messages/new-message/?prescriptionId=25203727&redirectPath=%2Fmy-health%2Fmedications) username: karsyn.gpsysasynctwo@id.me

<img width="503" height="549" alt="Screenshot of Select a team radio buttons showing the error message" src="https://github.com/user-attachments/assets/462de899-14e3-48c3-b590-fc3cecd7cc1d" />

##### 🔗 From [Secure Messages > Select a care team](https://staging.va.gov/my-health/secure-messages/new-message/?prescriptionId=25203727&redirectPath=%2Fmy-health%2Fmedications) username: karsyn.gpsysasynctwo@id.me

<img width="739" height="502" alt="Screenshot of Select a team combobox showing the error message" src="https://github.com/user-attachments/assets/7ecc78e2-799a-42f0-b663-a2297d5418e7" />

---

#### 👣 Steps to recreate
1. Navigate to Care teams you recently sent messages to](https://staging.va.gov/my-health/secure-messages/new-message/?prescriptionId=25203727&redirectPath=%2Fmy-health%2Fmedications) username: karsyn.gpsysasynctwo@id.me 
2. Using a screen reader, use the Tab key to navigate to the Continue button without selecting a radio button option.
3. Use the Enter key to activate the Continue button.
4. Notice that the error message and error formatting displays.
5. Notice that focus is moved to the first radio button.
6. Notice that the first radio button label is announced and the control label is announced.
7. Notice that the error message is not announced.

1. Navigate to Select care team](https://staging.va.gov/my-health/secure-messages/new-message/?prescriptionId=25203727&redirectPath=%2Fmy-health%2Fmedications) username: karsyn.gpsysasynctwo@id.me 
2. Using a screen reader, use the Tab key to navigate to the Continue button without selecting an option from the combobox input.
3. Use the Enter key to activate the Continue button.
4. Notice that the error message and error formatting displays.
5. Notice that focus is moved to the combobox input.
6. Notice that the combobox input label is announced.
7. Notice that the error message is not announced.

</details>

---

### 🔗 Point of contact
Sarah Horton (@sarahhortonadhoc)

### ⚠️ Severity level
- 2, Serious. Should be fixed in 1-2 sprints post-launch.

#### Accessibility Standards
- [Guideline 4.1 Compatible: Maximize compatibility with current and future user agents, including assistive technologies](https://www.w3.org/WAI/WCAG21/Understanding/compatible)
  - [SC 4.1.3 Status Messages (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html)

#### Experience Standards
- [Category Usability: User is able to easily know or learn how to use the interface to complete tasks](https://design.va.gov/about/experience-standards/#category-usability)
  - [User can identify an element and its state](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)

### 𝌣 Proposed solution or next steps
- Add an aria-describedby attribute to the input that references the id of the error message (`id="radio-error-message"`)

### 🔎 Supporting resources
- [Form error handling](https://design.va.gov/components/form/#error-handling)

### ✅ Acceptance criteria
- [ ] The system provides an error message when the user selects Continue without completing the input.
- [ ] The screen reader announces the error message along with the input label.
- [ ] The user can see and hear the error message when the select Continue without completing the input.

### ✅ Possible tasks
- [x] Review, refine and acknowledge feedback.
- [x] Update ticket information (assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

### 🔎 General resources
- USWDS Docs: [USWDS Documentation](https://designsystem.digital.gov/)
- VADS Docs: [VADS Experience Standards](https://design.va.gov/about/experience-standards/)
- VADS Docs: [VADS Components Documentation](https://design.va.gov/components/)
- VADS Docs: [VADS Components Storybook](https://design.va.gov/storybook/?path=/docs/about-introduction--docs)
- VA Platform Docs: [Accessibility defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric)
- VA Github Docs: [List of common issue tickets](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/accessibility/common-issues) you can reference, copy, and add to.
- W3C Docs: [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/)
- W3C Docs: [WCAG 2.2 Understanding Docs](https://www.w3.org/WAI/WCAG22/Understanding/)

**Comment by sarahhortonadhoc** [comment]
@amyleadem thank you for checking and for your guidance! @sofia-cordero @steveg-IAT, see [Amy's comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/124531#issuecomment-3807812689), is this something you can take forward?
